genie/transform.py

```diff
@@ -64,3 +65,24 @@ def _convert_df_with_mixed_dtypes(read_csv_params: dict) -> pd.DataFrame:
         df = pd.read_csv(**read_csv_params, low_memory=False, engine="c")
     warnings.resetwarnings()
     return df
+
+
+def _convert_values_to_na(
+    input_df: pd.DataFrame, values_to_replace: list, columns_to_convert: list
+) -> pd.DataFrame:
+    """Converts given values to NA in an input dataset
+
+    Args:
+        input_df (pd.DataFrame): input dataset
+        values_to_replace (list): string values to replace with na
+        columns_to_convert (list): subset of columns to convert with na in
+
+    Returns:
+        pd.DataFrame: dataset with specified values replaced with NAs
+    """
+    if not input_df.empty:
+        replace_mapping = {value: None for value in values_to_replace}
```

Comment:
So, I'm seeing [this in maf.py's _validation](https://github.com/Sage-Bionetworks/Genie/pull/549/files#diff-323044b86c877fd4af48462d3e131927c1a76d6ecef14abf031d1847a7619718L199-R214) in which this is handled (originally). I'm not seeing anything in processing in annotation-tools for maf.py that resembles what is happening in validation. Which is surprising, because I would think we'd want to do the same?

I think I have to move the code from above specifically to the [_convert_values_to_na in genie/transform.py](https://github.com/Sage-Bionetworks/Genie/pull/549/files#diff-f5e09cd8a62285bf7d3ff1917b948de18ec663c78a15a600176178682757fd8eR70) method just to make sure it's int/float dtypes for both validation and processing

Reply:
After experimenting, I think having `None` works. A dataframe with string values and None (e.g: `pd.DataFrame({"bye":["2314", None, "124"]})`) converted to float will convert the column to float just fine.

Reply:
It seems like for `vcf` files' validation, we don't have to worry about this numeric stuff handling in validation or processing because `POS` is the only expected numeric column and it can't have NAs anyways.

Reply:
So most of the heavy lifting of processing is done via genome nexus and once we write it out into a csv, it doesn't matter what "type" the column is. 

That said - we do want to be diligent about making sure the data itself isn't changing, but a '1' and 1 is going technically be the same when you write it out.

Reply:
Ah I see, then I think it's okay as it is? I am for keeping `None` mainly because we can have more than just numeric columns